tests/data_files/Makefile

```diff
@@ -999,6 +999,57 @@ ec_bp512_pub.comp.pem: ec_bp512_pub.pem
 	$(OPENSSL) ec -pubin -in $< -out $@ -conv_form compressed
 all_final += ec_bp512_pub.comp.pem
 
+################################################################
+#### Convert PEM keys in DER format
+################################################################
+server1.pubkey.der: server1.pubkey
+	$(OPENSSL) pkey -pubin -in $< -out $@ -outform DER
+all_final += server1.pubkey.der
+
+rsa4096_pub.pem.der: rsa4096_pub.pem
```

Comment:
Hadn't noticed before, but I'm not super happy with `.pem.der`: that's kind of a contradiction, so I think it should be just `.der`. I don't think the fact the the DER files happen to have been generated from the PEM files matters enough to be reflected in the name, as it could just as easily been the other way round, and it wouldn't change the resulting files at all.

(Note: things like `.crt.pem` or `.key.pem` / `.key.der` make sense because they're like `what_thing.which_encoding`, but here that's not the case with `.pem.der`.)